src/scripts/rank_SVs.pl

```diff
@@ -363,6 +365,49 @@ sub is_dup_SV {
 	return 0;
 }
 
+sub exists_knownfusionlist_checking {
+        my %genelist = %{(shift)};
+        my $targetgene1 = shift;#e.g. targetgene UBTF,MIR6782
+        my $targetgene2 = shift;#e.g. targetgene UBTF,MIR6782
+        my @genes1 = split(/,|\|/, $targetgene1);
+        my @genes2 = split(/,|\|/, $targetgene2);
+
+        foreach my $g1 (@genes1) {
+                foreach my $g2 (@genes2){
+                        return 1 if(exists($known_fusions{$g1.":".$g2}));
+			return 1 if(exists($known_fusions{$g2.":".$g1}));
+                }
+        }
+
+        return 0;
+}
+
+sub exist_multiplename_checking {
+        my %genelist = %{(shift)};
+        my $targetgene = shift;#e.g. targetgene UBTF,MIR6782
+        my @genes = split(/,|\|/, $targetgene);
+
+        foreach my $g1 (@genes) {
+                return 1 if(exists($known_fusion_partners{$g1}));
+        }
+
+        return 0;
+}
+
+sub match_known_ITD {
```

Comment:
Why not just reuse a function to check if the element exists? Then you can just access the element directly, rather than returning the values.